apps/portal/.env.sample

```
@@ -1,7 +1,7 @@
 NEXT_PUBLIC_EXTENSION_NAME=Rec
 NEXT_PUBLIC_EXTENSION_ID=gneepehahiglangakfifnpdlppijdkck
 NEXT_PUBLIC_GOOGLE_CLIENT_ID=
-NEXT_PUBLIC_EXTENTION_REDIRECT_URL=
+NEXT_PUBLIC_EXTENSION_REDIRECT_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_EXTENSION_REDIRECT_URL key should go before the NEXT_PUBLIC_GOOGLE_CLIENT_ID key

(UnorderedKey)